doc/source/faq.rst

```diff
@@ -106,7 +442,6 @@ See `the NumPy documentation on byte order
 <http://docs.scipy.org/doc/numpy/user/basics.byteswapping.html>`__ for more
 details.
 
-
 Visualizing Data in Qt applications
```

Comment:
This section should be removed and we should simply add a mention of pandas-qt to the "pandas ecosystem" page is appropriate.